packages/node/src/resolve.js

```diff
@@ -69,52 +72,59 @@ export const resolveEntrypoint = (specifier, from = process.cwd()) => {
 }
 
 /**
- * Resolve a path to a bin script in the closest `node_modules/.bin/` dir.
+ * Resolve a path to an executable ("bin" script) in the closest
+ * `node_modules/.bin/` dir.
  *
  * @remarks
  * This is not recursive due to the potential for stack overflows (though
- * unlikely).
+ * unlikely). TODO: refactor to be async
  * @param {string} name Bin script name
  * @param {ResolveBinScriptOptions} [options]
  * @returns {string} Path to the bin script
  */
-export const resolveBinScript = (
+export const resolveExecutable = (
```

Comment:
Bin has fewer alternative meanings than Executable. This is not an improvement IMHO

Reply:
my reason for changing it was that "bin" is short for "binary" which these are not. but yes, it's really lost all meaning since any sort of executable could live in a `bin` dir.

I'll change it back.